hypha/apply/projects/templates/application_projects/project_admin_detail.html

```diff
@@ -4,14 +4,24 @@
 
 
 {% block admin_assignments %}
-    {% if object.is_in_progress %}
+    {% user_can_update_project_status object user as can_update_status %}
+    {% if can_update_status %}
         <div class="js-actions-sidebar sidebar__inner sidebar__inner--light-blue sidebar__inner--actions {% if mobile %}sidebar__inner--mobile{% endif %}">
 
             <h5>{% trans "Actions to take" %}</h5>
 
-            {% user_can_update_project_status project user as can_update_status %}
+            {% user_can_skip_pafapproval_process project user as can_skip_paf %}
+            {% if can_skip_paf %}
+                <a data-fancybox data-src="#continue-to-next-phase" class="button button--white button--full-width button--bottom-space" href="#">{% trans "Continue to next status" %}</a>
+
+                <div class="modal" id="continue-to-next-phase">
+                    <h4 class="modal__project-header-bar">{% trans "Continue to next stage" %}</h4>
+                    <p>{% trans "Have you filled the PAF and sure to move to the next stage, this action can not be reverted" %}</p>
```

Comment:
What are your thoughts on the verbiage here being changed to something like `Please ensure the Project Form is completed and you are ready to proceed to the next stage. This action cannot be reverted.` for a bit more clarity & to use the new "Project Form" vs "Project Approval Form" language?

Reply:
Updated the text now.